operator/controller/src/main/deploy/examples/minikube.apicurioregistry3.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: registry.apicur.io/v1
+kind: ApicurioRegistry3
+metadata:
+  name: minikube-apicurioregistry3
+spec:
+  app:
+    ingress:
+      host: registry-api.apps.minikube.local
+    features:
+      allowDeletes: true
+  ui:
+    ingress:
+      host: registry-ui.apps.minikube.local
```

Comment:
This example file(s) are those that show up as defaults when a new CR is created in OCP console. There is another example directory at `operator/controller/src/test/resources/k8s/examples` which is used for more general showcase of features, to be used in docs and install examples. It is located in the test directory because we can also use the examples during testing. Can you please move it there and rename it, since `minikube` is a very generic name.